lib/datadog/tracing/contrib/grape/endpoint.rb

```diff
@@ -101,6 +101,9 @@ def endpoint_run(name, start, finish, id, payload)
 
                 span.set_tag(Tracing::Metadata::Ext::HTTP::TAG_METHOD, request_method)
                 span.set_tag(Tracing::Metadata::Ext::HTTP::TAG_URL, path)
+
+                # TEMP REMOVE ONCE SENT TO RACK // ENSURE APPLICATION ROOT IS PREPENDED IN RACK VIA URL#
```

Comment:
To accomplish this, I suggest adding the Grape route to the active Rack `env`.

This way the Rack integration can read that `env` value and set the tag appropriately.